### PR TITLE
Release v1.6.0

### PR DESCRIPTION
**Pull request by bg451**

### CODE REVIEW

**Comment by bg451** [comment]
* **#24** <a href="https://app.graphite.dev/github/pr/chronosphereio/chronoctl-core/24?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @bg451 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 
-## Unreleased
+## v1.6.0
 
+## Added
 * Added support for entity namespaces
 * Added support for datasets
 * unstable: Added support for otel-metrics-ingestion
-* Promote gcp-metrics-integration to stable
+* Promotes `unstable/config/GcpMetricsIntegration` to `v1/config/GcpMetricsIntegration`
```
Comment (aschepis) [inline CHANGELOG.md:9]: can we reword this in terms of `chronoctl` behavior?  does a command change? if not then we can probably just put it as Internal

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,12 @@
 # Releases
 
-## Unreleased
+## v1.6.0
 
+## Added
 * Added support for entity namespaces
 * Added support for datasets
 * unstable: Added support for otel-metrics-ingestion
```
Comment (prashantv) [inline CHANGELOG.md:8]: mind adding the `kind` for otel-metrics-ingestion and datasets as well

**Review by prashantv — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (bg451) [reply CHANGELOG.md]: Ope let me repush this with the latest change. Unstable was not released, so effectively we're just adding this to the primary command list.

**Pull request merged**